pkg/virt-controller/watch/vm_test.go

```diff
@@ -427,48 +447,83 @@ var _ = Describe("VirtualMachine", func() {
 			Expect(vm.Status.PrintableStatus).To(Equal(v1.VirtualMachineStatusProvisioning))
 
 		})
+		Context("Disk un/hotplug", func() {
+			addVolumeReactor := func(virtFakeClient *fake.Clientset) {
```

Comment:
It will be good, in the future, to have these reactors available by default.
IOW this behavior should be provided by default by the fakeclient, and we should only care when diverge from standard behavior.

Reply:
I think it should be enough for this context but I can take this as follow up

Reply:
Yeah, sorry! I mean as a follow up, not in this PR, my bad